LEGO1/lego/legoomni/include/legoinputmanager.h

```diff
@@ -7,6 +7,7 @@
 #include "mxpresenter.h"
 #include "mxqueue.h"
 
+#include <SDL3/SDL_joystick.h>
 #include <SDL3/SDL_keyboard.h>
 #include <dinput.h>
```

Comment:
Maybe we can remove this Windows header now?

Reply:
I had the same thought, but it unfortunately seems to fail due to `MxResult Create()` taking an `HWND` as a parameter. Assuming this is inherited recursively, we may be able to replace `dinput.h` with `windows.h` here since we aren't using any DirectInput-specific code anymore.

Reply:
Yes. Let's replace it with `windows.h` then!

Reply:
Hadn't seen your edit until now for some reason, glad we had the same idea!